lib/hyperion_test/fake_server/types.rb

```diff
@@ -0,0 +1,7 @@
+class Hyperion
+  class FakeServer
+    MimicRoute = ImmutableStruct.new(:method, :path)
+    Rule = ImmutableStruct.new(:mimic_route, :headers, :handler, :rest_route)
+    Request = ImmutableStruct.new(:body)
```

Comment:
RubyMine flags these as not using the 'constant' style of screaming case.

MIMIC_ROUTE
RULE
REQUEST

Is there a reason these constants are not screaming case? I have seen this in ascent-web and assaymatic as well.